syntax = "proto3";

package basicmessage;

service BasicService {
  rpc ShowMessage (SimpleMessage) returns (OutputMessage) {}
  rpc DetailMessage (SimpleMessage) returns (OutputMessage) {}
}

service SecondService {
  rpc TranslateMessage (SimpleMessage) returns (OutputMessage) {}
  rpc BuildMessage (SimpleMessage) returns (OutputMessage) {}
}

message SimpleMessage {
  string name = 1;
}

message OutputMessage {
  string message = 1;
}
